src/components/pages/species/list/views/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import useSpeciesList from "../use-species-list";
+import GridView from "./list-tiles";
+import ListView from "./table";
+
+export default function Views() {
+  const {
+    filter: { f }
+  } = useSpeciesList();
+
+  switch (f?.view) {
+    case "list":
+      return <ListView />;
```

Comment:
looking at condition this two lines can be safely removed or this could've just been if return and `else` (in this case can be omited return)